inst/starter_models/seir/model.R

```diff
@@ -0,0 +1,33 @@
+library(macpan2)
+
+## expression lists
+###################################################
+
+## free form computations for convenience
+computations = list(
+  N ~ sum(S, E, I, R)
+)
+
+## absolute flow rates (per time only)
+flow_rates = list(
```

Comment:
I'm no epidemiologist, so please correct me if I am wrong, but I prefer the word `progression` for what is described below as `exposure`. The rate `alpha * E` is the rate of flow from individuals who are infected but not infectious (`E`) to individuals who are both infected and infectious (`I`). The exposure has already taken place, and what is happening is that the disease case is _progressing_ from a state where the individual is not infectious to one where it is.

Reply:
I think we've also talked about this before - AFAIK there's no established terminology for this, "progression" is probably as good as anything ("end of the latent period/commencement of infectivity")? I might ask on Zulip ...